Modules/Main.lua

```diff
@@ -482,12 +483,22 @@ function main:LoadSettings(ignoreBuild)
 				if node.attrib.nodePowerTheme then
 					self.nodePowerTheme = node.attrib.nodePowerTheme
 				end
-				if node.attrib.showThousandsSidebar then
-					self.showThousandsSidebar = node.attrib.showThousandsSidebar == "true"
-				end -- else leave at default
-				if node.attrib.showThousandsCalcs then
-					self.showThousandsCalcs = node.attrib.showThousandsCalcs == "true"
-				end -- else leave at default
+				-- In order to preserve users' settings through renameing/merging this variable, we have this if statement to use the first found setting
+				-- Once the user has closed PoB once, they will be using the new `showThousandsSeparator` variable name, so after some time, this statement may be removed
+				if node.attrib.showThousandsCalcs ~= nil then
+					self.showThousandsSeparators = node.attrib.showThousandsCalcs == "true"
+				elseif node.attrib.showThousandsSidebar ~= nil then
+					self.showThousandsSeparators = node.attrib.showThousandsSidebar == "true"
+				end
```

Comment:
Shouldn't need the `nil` comparison here

```suggestion
				if node.attrib.showThousandsCalcs then
					self.showThousandsSeparators = node.attrib.showThousandsCalcs == "true"
				elseif node.attrib.showThousandsSidebar then
					self.showThousandsSeparators = node.attrib.showThousandsSidebar == "true"
				end
```